doc/apiary/v2/fiware-ngsiv2-cookbook.apib

```diff
@@ -397,16 +397,18 @@ This resource allows you to obtain a list of subresources published by the API:
 
 ## Query Entities [/v2/entities{?q,limit,offset,coords,geometry,attrs,options}]
 
-### Value filter and Geoquery [GET /v2/entities{?q,limit,offset,coords,geometry,attrs,options}]
+### Value filter and Geoquery [GET /v2/entities{?q,limit,offset,coords,georel,geometry,attrs,options}]
```

Comment:
fixed https://github.com/telefonicaid/fiware-orion/pull/1959/commits/f3fd76c3817fd1a5b8f5d1059e37ebd4be19f371 

the example has been modified appropriately